src/kakarot/account.cairo

```diff
@@ -650,6 +689,26 @@ namespace Account {
         );
         return self;
     }
+
+    func compute_code_hash{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
actually - we should have a test for this

Reply:
Done with [ea35f94](https://github.com/kkrt-labs/kakarot/pull/1309/commits/ea35f9463c2bc38432bf3b7232f26cf726935a4f)

Reply:
Fixed with [ed6c1db](https://github.com/kkrt-labs/kakarot/pull/1309/commits/ed6c1db86fb3c1d37c65fb15e34d46275a698652)
`cairo_run` interpret the return as a int and do not take into account the leading 0
https://github.com/kkrt-labs/kakarot/actions/runs/10112754238/job/27967569334